lib/rspec/abq/manifest.rb

```diff
@@ -12,10 +12,12 @@ def self.write_manifest(ordered_groups, random_seed, registry)
       # @param ordered_groups [Array<RSpec::Core::ExampleGroup>] ordered groups to assemble into a manifest
       def self.generate(ordered_groups, random_seed, registry)
         {
+          type: "manifest_success",
           manifest: {
             init_meta: RSpec::Abq::Ordering.to_meta(random_seed, registry),
             members: ordered_groups.map { |group| to_manifest_group(group) }.compact
-          }
+          },
+          other_errors: nil
```

Comment:
🙏 should we pass an empty array for `other_errors`

Reply:
It's optional in the protocol, so I left it as nil so that on the abq side we avoid allocating an array to hold the zero errors